frontend/packages/integration-tests-cypress/views/timestamp.ts

```diff
@@ -0,0 +1,3 @@
+export const timestamp = {
+  isLoaded: () => cy.get('[data-cy=timestamp]').should('exist'),
```

Comment:
This should be `data-test` per [our best practices](https://github.com/openshift/console/blob/master/frontend/packages/integration-tests-cypress/README.md#migrating-protractor-tests-to-cypress) and `cy.byTestID('timestamp').should('exist')`